compiler/src/iree/compiler/Dialect/Stream/Transforms/EncodeTensors.cpp

```diff
@@ -465,6 +445,12 @@ struct EncodeTensorFillOp
       return failure();
     }
 
+    auto pattern = canonicalizeFillPattern(op.getValue(), rewriter);
+    if (!pattern) {
+      return rewriter.notifyMatchFailure(
```

Comment:
and here ("why unsupported? what is supported?")

Reply:
Updated to be more clear.